### PR TITLE
Move abi and contract data to global DB

### PR DESCRIPTION
**Pull request by LefterisJP**
- Moved ABI and contracta data to global DB
- Adds a new GlobalDB upgrade for this
- Removes old unused json files

### CODE REVIEW

**Review by yabirgb — COMMENTED** [review]
The files path were removed from the two files (`setup.py` and `.spec`) that is what we always forget. Just small comments

File: rotkehlchen/globaldb/upgrades/v3_v4.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    Also making sure to not repeat existing abis. Ran a script to determine which
+    abis are common between all contracts and encoding the output of these relations
+    to the upgrade.
+
+    eth_abi.json has no repeating ABIs
+    """
+    with connection.write_ctx() as cursor:
+        _create_new_tables(cursor)
+        _add_eth_abis_json(cursor)
+        _add_eth_contracts_json(cursor)
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/globaldb/upgrades/v3_v4.py:222]: Good point. Will do it.

File: rotkehlchen/globaldb/upgrades/v3_v4.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+def _get_or_create_common_abi(
+        cursor: 'DBCursor',
+        name: str,
+        serialized_abi: str,
+) -> int:
+    cursor.execute(
+        'SELECT id FROM contract_abi WHERE name=?',
+        (name,),
+    )
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/globaldb/upgrades/v3_v4.py:98]: not sure it fit

File: rotkehlchen/globaldb/upgrades/v3_v4.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    'YEARN_ASUSD_VAULT',
+    'YEARN_USDP_3CRV_VAULT',
+]
+
+
+def _get_abi(cursor: 'DBCursor', name: str) -> int:
+    cursor.execute(
+        'SELECT id FROM contract_abi WHERE name=?',
+        (name,),
+    )
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/globaldb/upgrades/v3_v4.py:72]: > The files path were removed from the two files (`setup.py` and `.spec`) that is what we always forget. Just small comments

Anything you mean here? Did I forget something? Or you just saying I remembered and it's all good?

[the remark this answers is not in the document]
Comment (yabirgb) [reply rotkehlchen/globaldb/upgrades/v3_v4.py:72]: Yes. Just mentioning that I checked it and is good.

**Comment by LefterisJP** [comment]
Frontend e2e tests failed due to the usual timeout
Single failed backend test is covalent timeout.

Merging.

**Pull request merged**